client/anp/index.ts

```diff
@@ -1,4 +1,13 @@
 import photo from './photo';
+import {startApp} from 'superdesk-core/scripts/index';
+import planningExtension from 'superdesk-planning/client/planning-extension/dist/src/extension';
+
+setTimeout(() => {
```

Comment:
why it needs the timeout?

Reply:
To work around circular dependency. client-core is importing this via `importApps` and I need to import `startApp` from client core. As we move stuff to extensions, we'll remove `importApps` someday, but until then a timeout is needed.

Reply:
so the import is fine but you can't call that function right away?

Reply:
Yep.